skt/runner.py

```diff
@@ -98,6 +122,15 @@ def dumpjunitresults(self, jobid, junit):
             (stdout, stderr) = bkr.communicate()
 
     def getconsolelog(self, jobid=None):
+        """
+        Retrieve console log from a Beaker job.
```

Comment:
I think the description should say "log URL". Otherwise this is confusing, as e.g. I assumed this would return the logs themselves. We also need to fix the function name in a separate commit (not necessarily now).

Reply:
Fixed.